packages/babel-preset-expo/babel.config.js

```diff
@@ -6,6 +6,6 @@ module.exports = function (api) {
       // Ignore tests and the `unknown` name used in tests
       return !filename.match(/samples/) && filename.match(/src\/(.*)\.tsx?/);
     },
-    presets: ['@expo/babel-preset-cli'],
+    presets: ['babel-preset-expo'],
```

Comment:
babel-preset-expo is designed to be used with a bundler like Webpack or Metro, it's not intended for use with Node.js directly (at the moment). Something more focused could be a better fit here.

Reply:
Yep, I'm opening up a new PR that adds a "node" specific babel target inside `expo-module-scripts`, so we can still use it for the CLI packages. I wonder if we need to update the `plugin` to also reuse this node preset, as that's built for node as well right? @EvanBacon

Reply:
https://github.com/expo/expo/pull/25458